### PR TITLE
Create subzero.yml

### PR DESCRIPTION
**Pull request by superiorxman**

### CODE REVIEW

**Comment by superiorxman** [comment]
http://gamersprey.com/wp-content/uploads/Injustice-2-Sub-Zero.jpg
freezing/ice powers

**Comment by superiorxman** [comment]
http://assets.vg247.com/current/2010/11/MK9_360_SubZeroVsCage_GroundFreeze_LivingForest_WEB.jpg
more ice powers

**Comment by superiorxman** [comment]
https://assets.vg247.com/current/2017/07/injustice_2_sub_zero_1.jpg
can you also give him an ice sword

File: CommandHelper/LocalPackages/commands/chars/subzero.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  val: eyJ0aW1lc3RhbXAiOjE1MjMxNjMzNDM2MzAsInByb2ZpbGVJZCI6ImVmN2JjYzk2M2FjMDRkYzZhYWU3MDE5ZTVjNmRkYTJlIiwicHJvZmlsZU5hbWUiOiJzdXBlcmlvcnhtYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzlhNTk5MGFkNGNiYjVlZTNlNzQ1YjYzOWY4MjM0ODQzODY3MzQyYjlmNDZmNWEyMDJhODNkY2QyYmM1ODUxIn19fQ==
+effects:
+- /effect <plr> 10 9999 2 true
+- /effect <plr> 11 9999 1 true
+items:
+- /shot give <plr> icebeam
+- /give <plr> minecraft:ice 5 16:10 17:10 name:&9J&1a&9ck_&fF&9r&bo&9st&8's(&bM&8)_&9I&1c&9e&9_F&1i&9s&1t
```
[the remark this answers is not in the document]
Comment (malonnnn) [reply CommandHelper/LocalPackages/commands/chars/subzero.yml]: This is true

File: CommandHelper/LocalPackages/commands/chars/subzero.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- /give <plr> minecraft:iron_boots 1 9:3 name:&9J&1a&9ck_&fF&9r&bo&9st&8's(&bM&8)_&9B&3o&8ot&fs
```
[the remark this answers is not in the document]
Comment (malonnnn) [reply CommandHelper/LocalPackages/commands/chars/subzero.yml]: This is also true

File: CommandHelper/LocalPackages/commands/chars/subzero.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  val: eyJ0aW1lc3RhbXAiOjE1MjMxNjMzNDM2MzAsInByb2ZpbGVJZCI6ImVmN2JjYzk2M2FjMDRkYzZhYWU3MDE5ZTVjNmRkYTJlIiwicHJvZmlsZU5hbWUiOiJzdXBlcmlvcnhtYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzlhNTk5MGFkNGNiYjVlZTNlNzQ1YjYzOWY4MjM0ODQzODY3MzQyYjlmNDZmNWEyMDJhODNkY2QyYmM1ODUxIn19fQ==
+effects:
+- /effect <plr> 10 9999 2 true
+- /effect <plr> 11 9999 1 true
+items:
+- /shot give <plr> icebeam
+- /give <plr> minecraft:ice 5 16:10 17:10 name:Subzero's Ice Fist
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/subzero.yml]: requires underscores between the words

**Pull request merged**